runtime/list_test.go

```diff
@@ -392,22 +392,20 @@ func TestListSetItem(t *testing.T) {
 }
 
 func TestListSort(t *testing.T) {
-	fun := newBuiltinFunction("TestListSetItem", func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
-		if raised := checkFunctionArgs(f, "TestListSetItem", args, ListType); raised != nil {
-			return nil, raised
-		}
-		l := toListUnsafe(args[0])
-		if raised := l.Sort(f); raised != nil {
+	fun := newBuiltinFunction("TestListSort", func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+		if _, raised := listSort(f, args, KWArgs{}); raised != nil {
```

Comment:
Updated to reflect that. 

One question, so we are testing 
```
func (l *List) Sort(f *Frame) (raised *BaseException) {
```

Instead of 
```
func listSort(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
```

Correct?